client/src/app/tabs/dmn/DmnEditor.js

```diff
@@ -143,6 +145,12 @@ export class DmnEditor extends CachedComponent {
     const initialView = modeler._getInitialView(modeler._views);
 
     this.open(initialView.element);
+
```

Comment:
Drop check and simply execute `onImport(null, warnings)`?

Reply:
👍